config/prod.exs

```diff
@@ -6,6 +6,14 @@ config :bike_brigade, BikeBrigadeWeb.Endpoint,
   cache_static_manifest: "priv/static/cache_manifest.json",
   force_ssl: [rewrite_on: [:x_forwarded_proto], hsts: true]
 
+config :bike_brigade, BikeBrigade.Messaging.Slack.Operations,
```

Comment:
I think we should have staging use the api-playground channel as well - can you move this into `runtime.exs`, there's an if statement that decides whether its staging or production 

If its unclear the config stuff is kind of dodgy - prod.exs is when the compile-time environment is production which is true for staging and production. runtime.exs can decide what the runtime env is, which is set with an environment variable (see https://github.com/bikebrigade/dispatch/blob/main/fly.toml#L6-L7 vs https://github.com/bikebrigade/dispatch/blob/main/fly.staging.toml#L6-L7 for how its set)

Reply:
Gotcha! Ya I was a bit confused between `runtime`/`config`! Will move.